### DISCUSSION TITLE
Apollo template version 8.0.1 compability with Prime react version?

### DISCUSSION THREAD

**Discussion by RamaKrishna-CEVA** [primefaces/community#349] [PrimeReact] [answered]
We are using the Apollo template (Prime React version) 8.0.1 (CRA version) we noticed the new version of Apollo was migrated from CRA to next.js, we have a huge number of applications that already use the Apollo template built on CRA, not Next.js. migration our application to use Apollo with next.js is going to cost a lot of effort and is going to re-restructure our applications,

1. Do you have another template close to Apollo that is still working and supports the latest version of Primereact and CRA?
2. Does PrimeReact Appollo support the latest version of PrimeReact
3. Do you have any other idea

**Comment by gucal** [comment, marked as answer]
Hi,

Unfortunately, our CRA support for Apollo has ended. New versions can only be used with Next.js. 
Apollo versions are currently available for PrimeReact 9.x versions. Version 10 is not yet supported. Will support in the coming days. 

Thank you.
